libsel4/Kconfig

```diff
@@ -16,24 +16,35 @@ menu "libsel4"
         help
             seL4 API library
 
+choice
+    prompt "Function attributes"
+    default LIB_SEL4_DEFAULT_FUNCTION_ATTRIBUTES
```

Comment:
I think the default should be `LIB_SEL4_INLINE_INVOCATIONS`, to match that it was previously `default y`?